lib/common/formats.js

```diff
@@ -12,13 +12,19 @@
  */
 
 var _ = require('lodash');
-var module_def = 'module.exports = ';
 
-function fileHeader() {
-  return '/**\n' +
-         ' * Do not edit directly\n' +
-         ' * Generated on ' + new Date() + '\n' +
-         ' */\n\n';
+function fileHeader(options) {
+  var to_ret = '';
+  // for backward compatibility we need to have the user explicitly hide them
+  var showHeader = (options) ? options.showHeader : true;
```

Comment:
What are your thoughts about renaming this? I'm thinking if we eventually want options to cascade (root > platform > file), we might want this option name to be more explicit it is about files. Open to suggestions, maybe showFileComment ?

Reply:
I went for `showFileHeader`. Do you think it's clear enough?